saemref/install.sls

```diff
@@ -41,3 +41,48 @@ cubicweb-create:
     - require:
         - pkg: cube-packages
         - user: {{ saemref.instance.user }}
+
+{% if saemref.instance.wsgi %}
+
+{% if not saemref.instance.cookie_secret_key %}
+cookie_secret_key_must_be_set:
+  cmd.run:
+    - name: echo "cookie_secret_key must be set" && false
+    - failhard: true
+{% endif %}
+
+wsgi-packages:
+  pkg.installed:
+    - pkgs:
+      - pyramid-cubicweb
+      - uwsgi
+      - uwsgi-plugin-python
+    {% if grains['os_family'] == 'Debian' %}
+      - redis-server
+      - python-pyramid-redis-sessions
+    - skip_verify: true {# FIXME: key expired... #}
+    - require:
+      - pkgrepo: logilab-backports
+      - pkgrepo: backports
+    {% else %}{# RedHat #}
+      - redis
+      - pyramid_redis_sessions
+    - require:
+      - pkgrepo: logilab_extranet
+    {% endif %}
+
+redis-server-service-running:
+{% if is_docker_build %}
+{#
+Salt fail to enable a systemd service if systemd is not running (during the
+docker build phase) This is a workaround.
+#}
```

Comment:
I think it'd be nice to not need this by running state.highstate outside in a running container instead of in Dockerfile. I propose to handle this in a future PR.